blueprint/index.md

```diff
@@ -1,224 +1,250 @@
 * REST API authentication
 * Genesys Cloud scripting
 
-## Genesys Cloud account required
+### Genesys Cloud account
+
+* A Genesys Cloud 3 license with agentless SMS functionality. For more information, see [Genesys Cloud Pricing](https://www.genesys.com/pricing "Opens the pricing article").
+* The Master Admin role in Genesys Cloud. For more information, see [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article") in the Genesys Cloud Resource Center.
+
+### Microsoft account
 
-This solution requires the SMS functionality included with a Genesys Cloud 3 license. For more information on licensing, see [Genesys Cloud Pricing](https://www.genesys.com/pricing "Opens the pricing article").
+* Admininstrator-level role for Microsoft Azure Active Directory to set up authorization and grant permissions for Genesys Cloud. 
+* Microsoft Teams license for agents.
 
-A recommended Genesys Cloud role for the solutions engineer is Master Admin or Script Designer. These roles include the required permission: Scripter > All > All. For more information on Genesys Cloud roles and permissions, see the [Roles and permissions overview](https://help.mypurecloud.com/?p=24360 "Opens the Roles and permissions overview article").
+## Implementation steps
 
-## Microsoft account required
-The solution requires administrator level credentials for Microsoft Azure Active Directory that allow you to set up authorization and grant permissions for Genesys Cloud. Agents require the licensed use of Microsoft Teams.
-# Deployment steps
+  - [Configure Microsoft Azure custom app](#configure-microsoft-azure-custom-app)
+  - [Configure Genesys Cloud](#configure-genesys-cloud)
+  - [Add a web services data actions integration](#add-a-web-services-data-actions-integration)
+  - [Create an OAuth client for use with the Genesys Cloud data action integration](#create-an-oauth-client-for-use-with-the-genesys-cloud-data-action-integration)
+  - [Add a Genesys Cloud data actions integration](#add-a-genesys-cloud-data-actions-integration)
+  - [Load Supporting Data Actions](#load-supporting-data-actions)
+  - [Import Create Teams Video Meeting data action](#import-create-teams-video-meeting-data-action)
+  - [Send SMS data action](#send-sms-data-action)
+  - [Load Script](#load-script)
 
-## Configure Microsoft Azure Custom App
-You must register your custom application within Azure to enable the Genesys Cloud instance to authenticate and retrieve user information from the Graph API for Teams.
+### Configure Microsoft Azure custom app
+To enable the Genesys Cloud instance to authenticate and retrieve user information from the Graph API for Microsoft Teams, register your custom application in Azure. 
 
-1. Log in to Azure, navigate to **App Registrations**, and click **New Registration**.
+1. Log in to the Azure portal.
+2. Navigate to **App registrations**, and click **New registration**.
```

Comment:
Serial comma is not needed in a list of only 2 items. "Navigate to **App registrations** and click **New registration**."